### ISSUE TITLE
S3 bucket creation instructions slightly confusing

### ISSUE THREAD

**Issue by oudeismetis** [thinknimble/tn-cli#16]
Minor bit of confusion

The instructions say:
![image](https://github.com/user-attachments/assets/61ca7840-577f-4918-b19b-80136b050da4)

But in reality you need to call it like this:
```
tn aws-make-s3-bucket bucket-name thinknimble
```

Calling it with `profile='thinknimble'` results in a call like this:
```
... --capabilities CAPABILITY_NAMED_IAM --profile=profile=thinknimble
```

**Comment by whusterj** [comment]
I agree this is confusing... I like `just`'s approach, but its command documentation and tab completion functionality are rough. I think this is auto-generated by `just`, but I'll see if there's anything we can do to make it clearer.
